contribs/src/main/java/com/netflix/conductor/contribs/http/GenericHttpTask.java

```diff
@@ -145,6 +146,9 @@ HttpResponse httpCall(Input input, Task task, Workflow workflow, WorkflowExecuto
 		if (headers.containsKey(HttpHeaders.AUTHORIZATION)) {
 			headers.put(HttpHeaders.AUTHORIZATION, "xxxxxxxxxxxxxxxxxxx");
 		}
+		if (headers.containsKey("Authorization-Context")) {
+			headers.put("Authorization-Context", "xxxxxxxxxxxxxxxxxxx");
```

Comment:
Why are these values set to `x`s?

Reply:
@hueys  this seems like the string  builder for the log.
For the log, we don't want to log the real value but we want to log we got the auth token. so replace the value to "xxxxxx" was recommended from auth team.